### PR TITLE
Add columnar FileDecorations protobuf message definitions

### PR DESCRIPTION
**Pull request by schroederc**
These are foundational definitions for #2930.

A large part of this change is the move of the Node message to the schema package so that it can be used for file decoration target nodes.

### CODE REVIEW

File: kythe/go/serving/pipeline/nodes/nodes.go [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-// Package nodes provides Beam transformations over *ppb.Nodes.
+// Package nodes provides Beam transformations over *scpb.Nodes.
```
Comment (creachadair) [inline kythe/go/serving/pipeline/nodes/nodes.go]: To the reader in http://godoc.org/, the `scpb` won't denote anything; I suggest "…over `kythe.proto.schema.Node` messages." or words to that effect.

File: kythe/proto/schema.proto [hunk 2]
```diff
@@ -165,3 +167,56 @@ enum EdgeKind {
...
+  }
+}
+
+// A Fact represents a single Kythe graph node fact.
+message Fact {
+  kythe.proto.VName source = 1;
+
+  oneof name {
+    kythe.proto.schema.FactName kythe_name = 2;
+    string generic_name = 3;
```
Comment (creachadair) [inline kythe/proto/schema.proto]: Please document these.

File: kythe/proto/schema.proto [hunk 3]
```diff
@@ -165,3 +167,56 @@ enum EdgeKind {
...
+  bytes value = 4;
+}
+
+// An Edge represents a single Kythe graph edge.
+message Edge {
+  kythe.proto.VName source = 1;
+  kythe.proto.VName target = 2;
+
+  oneof kind {
+    kythe.proto.schema.EdgeKind kythe_kind = 3;
```
Comment (creachadair) [inline kythe/proto/schema.proto]: These too, please.

File: kythe/go/serving/pipeline/nodes/nodes.go (hunk 1 above)
Comment (schroederc) [reply kythe/go/serving/pipeline/nodes/nodes.go]: The package name does show up in the documentation: https://godoc.org/kythe.io/kythe/go/serving/pipeline/nodes#Kind.  I wanted to be consistent with the function types.

File: kythe/proto/schema.proto (hunk 2 above)
Comment (schroederc) [reply kythe/proto/schema.proto]: ✓

File: kythe/proto/schema.proto (hunk 3 above)
Comment (schroederc) [reply kythe/proto/schema.proto]: ✓

File: kythe/go/serving/pipeline/nodes/nodes.go (hunk 1 above)
Comment (creachadair) [reply kythe/go/serving/pipeline/nodes/nodes.go]: That makes sense. SGTM.

**Pull request merged**